javascript/modules/client/src/context.ts

```diff
@@ -144,6 +136,37 @@ export class Context {
     }
   }
 
+  useIpfs(
```

Comment:
Same as above, this should be an internal non-class-related function with a different name